filter_plugins/ceph-ansible.py

```diff
@@ -0,0 +1,13 @@
+from ansible import errors
+
+def only_one_true(a, *args):
```

Comment:
this is not the case, a is not in *args but this is a syntax thing with ansible and passing to a filter.

if the statement is as follows:

var1 | only_one_true(var2, var3)

then var1 = a and *args contains var2, var3. There are other ways to perform this check.